operator-common/src/main/java/io/strimzi/operator/common/ShutdownHook.java

```diff
@@ -5,19 +5,26 @@
 package io.strimzi.operator.common;
 
 import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import io.vertx.core.Promise;
+import io.vertx.core.Verticle;
 import io.vertx.core.Vertx;
+import io.vertx.core.impl.Deployment;
+import io.vertx.core.impl.VertxImpl;
```

Comment:
Is this really something what should be used? Sounds not like a part of the public API.

Reply:
You are right. Let me see if there is a way to achieve the same using the public API.